test/dns_virsh/dns_virsh_test.rb

```diff
@@ -4,9 +4,8 @@
 
 class DnsVirshTest < Test::Unit::TestCase
   def test_virsh_provider_initialization
-    ::Proxy::SETTINGS.stubs(:virsh_network).returns('some_network')
-    server = Proxy::Dns::Virsh::Record.new
-    assert_equal "some_network", server.network
+    ::Proxy::Dns::Virsh::Plugin.settings.stubs(:network).returns('some_network')
```

Comment:
Sure, although I think it's useless. Changing a default value is indeed intrusive and it's quite obvious (without a test).